fxa/tests/__init__.py

```diff
@@ -1,3 +1,7 @@
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this file,
 # You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+from requests.packages import urllib3
+urllib3.disable_warnings()
```

Comment:
According to the documentation you are right: https://urllib3.readthedocs.org/en/latest/security.html#insecureplatformwarning

Maybe we can setup PyOpenSSL usage for PyFxA as described here: https://urllib3.readthedocs.org/en/latest/security.html#pyopenssl